docs/source/reference/rest.md

```diff
@@ -53,9 +60,19 @@ it for the given user with the Hub's database.
 In [version 0.8.0](../changelog.md), a token request page for
 generating an API token is available from the JupyterHub user interface:
 
-![Request API token page](../images/token-request.png)
+```{figure} ../images/token-request.png
+---
+name: token-request
+---
+JupyterHub's API token page
+```
 
-![API token success page](../images/token-request-success.png)
+```{figure} ../images/token-request-success.png
+---
+name: token-request-success
```

Comment:
I think we can lose the `name:` field. If we follow [the myst docs](https://myst-parser.readthedocs.io/en/latest/syntax/optional.html?highlight=figure#markdown-figures), we can use

```
:::{figure-md}

![token request success](../images/token-request-success.png)

JupyterHub's token page after successfully requesting a token.
:::